Bottom: 6fe0d9dfb0ad85a232bed9dc7110bfde2b412292
Top:    9ad5ad84d488b315357028e195dd5b062f57bed9
Author: Ian Kent <raven@themaw.net>
Date:   2013-08-19 17:59:02 +0800

Refresh of autofs-5.0.7-add-short-host-name-standard-marco-variable.patch

---

diff --git a/CHANGELOG b/CHANGELOG
index 3a3fec1..6e962f0 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -76,6 +76,7 @@
 - improve timeout option description.
 - only probe specific nfs version if requested.
 - fix bad mkdir permission on create.
+- add short host name standard marco variable.
 
 25/07/2012 autofs-5.0.7
 =======================
